Keas.Core/Domain/KeyData.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using Microsoft.EntityFrameworkCore;
+
+namespace Keas.Core.Domain
+{
+    public class KeyInfo
+    {
+        [Required]
+        public int id { get; set; }
+        
+        public Key Key { get; set; }
```

Comment:
List isn't returning a key. It is returning part of a key

Reply:
This should be fixed when I move it to the models folder